development/cpp/custom_resource_format_loaders.rst

```diff
@@ -57,55 +57,100 @@ read and handle data serialization.
 
 .. code:: cpp
 
-	#ifndef MY_JSON_LOADER_H
-	#define MY_JSON_LOADER_H
+    /* resource_loader_json.h */
 
-	#include "core/io/resource_loader.h"
+    #ifndef RESOURCE_LOADER_JSON_H
+    #define RESOURCE_LOADER_JSON_H
 
-	class ResourceFormatLoaderMyJson : public ResourceFormatLoader {
-	public:
-		virtual RES load(const String &p_path, const String &p_original_path, Error *r_error = NULL);
-		virtual void get_recognized_extensions(List<String> *p_extensions) const;
-		virtual bool handles_type(const String &p_type) const;
-		virtual String get_resource_type(const String &p_path) const;
+    #include "core/io/resource_loader.h"
+
+    class ResourceFormatLoaderJson : public ResourceFormatLoader {
+        GDCLASS(ResourceFormatLoaderJson, ResourceFormatLoader)
```

Comment:
C++ code is tab-indented in the docs, so this should be changed back.

Reply:
To be precise, it would be 4 spaces for the code block indentation, and then tabs for the actual C++ code.

Reply:
got it, done.